### ISSUE TITLE
AttributeError: 'set' object has no attribute 'items'

### ISSUE THREAD

**Issue by snugchaise** [rkistner/chinese-postman#14]
$ python postman.py --csv path.csv --gpx path.gpx test_graph.csv
Skipping input row 1
Traceback (most recent call last):
  File "postman.py", line 355, in <module>
    paths = chinese_postman_paths(component, n=5)
  File "postman.py", line 303, in chinese_postman_paths
    matchings = find_matchings(odd, n)
  File "postman.py", line 231, in find_matchings
    for u, v in best_matching.items():
AttributeError: 'set' object has no attribute 'items'`

**Comment by peternowee** [comment]
I can reproduce this error with NetworkX 2.2, but not with NetworkX 1.9 or 1.11. I suspect the error is caused by a change introduced in NetworkX 2.1.

Just before line 231, `postman.py` calls NetworkX's `max_weight_matching()`:
```
    best_matching = nx.max_weight_matching(graph, True)
    matchings = [best_matching]

    for u, v in best_matching.items():
```
NetworkX 2.1 changed the return type for `max_weight_matching()` from dictionary to set. Compare the documentation for [NetworkX 1.11 `max_weight_matching`](https://networkx.github.io/documentation/networkx-1.11/reference/generated/networkx.algorithms.matching.max_weight_matching.html) with [NetworkX 2.1 `max_weight_matching`](https://networkx.github.io/documentation/networkx-2.1/reference/algorithms/generated/networkx.algorithms.matching.max_weight_matching.html). See also https://github.com/networkx/networkx/pull/2774.

**Comment by rkistner** [comment]
Fixed in #18.
